magento/magento2
# Fix #9581 Add primary key on table catalog_url_rewrite_product_category

### Description (*)

This PR adds primary key on **catalog_url_rewrite_product_category** table. Missing primary key causes issues with Percona Xtradb cluster 5.7: https://www.percona.com/doc/percona-xtradb-cluster/LATEST/features/pxc-strict-mode.html#tables-without-primary-keys

### Fixed Issues (if relevant)
1. magento/magento2#9581: catalog_url_rewrite_product_category table missing PK

### Manual testing scenarios (*)

1. Run setup:upgrade on existing installation
2. Observe that primary key was added to catalog_url_rewrite_product_category table
3. Optional: if running percona cluster with "wsrep_certify_nonPK" off, setup:upgrade should not throw error

### Questions or comments
While it is possible to run Percona Cluster without primary keys with "wsrep_certify_nonPK" feature, having primary keys to every table is recommended approach. Based on my testing, catalog_url_rewrite is only such table that is missing the primary key.

### Contribution checklist (*)
